container-check

```diff
@@ -83,7 +83,7 @@ def populate_container_rpms_list((container)):
     return (subproc.returncode, container, rpms)
 
 
-def yum_update_container((container, name, packages)):
+def yum_update_container(container, name, packages):
```

Comment:
this broke py2 (which is still used in stable/queens)

Reply:
2021-06-02 18:54:15 | Traceback (most recent call last):
2021-06-02 18:54:15 |   File "/usr/bin/container-check", line 244, in <module>
2021-06-02 18:54:15 |     ret = list(p.map(yum_update_container, process_map))
2021-06-02 18:54:15 |   File "/usr/lib64/python2.7/multiprocessing/pool.py", line 250, in map
2021-06-02 18:54:15 |     return self.map_async(func, iterable, chunksize).get()
2021-06-02 18:54:15 |   File "/usr/lib64/python2.7/multiprocessing/pool.py", line 554, in get
2021-06-02 18:54:15 |     raise self._value
2021-06-02 18:54:15 | TypeError: yum_update_container() takes exactly 3 arguments (1 given)

Reply:
I think I missed to fix that part when I tried to fix the earlier invalid syntax error. I did propose a PR to notice that you've already done one.

2021-05-16 08:38:30 | File "/usr/local/bin/container-check", line 63
2021-05-16 08:38:30 | def populate_container_rpms_list((container)):
2021-05-16 08:38:30 | ^
2021-05-16 08:38:30 | SyntaxError: invalid syntax